src/app/workflow/versions/versions.component.html

```diff
@@ -15,170 +15,164 @@
-          *ngFor="let version of versions | orderBy: convertSorting(workflow?.mode)"
-          [ngClass]="{ info: _selectedVersion != null && version != null && _selectedVersion.id == version.id }"
+    <ng-container matColumnDef="snapshot">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        matTooltip="The metadata and descriptors for a version that has been snapshotted will not change over time."
+      >
+        Snapshot
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <mat-icon
+          *ngIf="!version.doiURL && version.frozen"
+          data-cy="dockstore-snapshot-locked"
+          matTooltip="Version {{ version.name }} is a snapshot."
+          >lock</mat-icon
+        >
+        <mat-icon
+          *ngIf="!version.doiURL && !version.frozen"
+          data-cy="dockstore-snapshot-unlocked"
+          matTooltip="Version {{ version.name }} is not a snapshot."
+          >lock_open</mat-icon
         >
-          <td>
-            <span *ngIf="publicPage && defaultVersion === version.name" class="label label-primary">Default</span>
-            <input
-              class="radio-button-reference"
-              *ngIf="version.name !== 'latest' && !publicPage"
-              type="radio"
-              name="defaultVersion"
-              [(ngModel)]="defaultVersion"
-              [value]="version.name"
-              (click)="updateDefaultVersion(version.name)"
-              [tooltip]="getDefaultTooltip(publicPage)"
-            />&nbsp;
-            <i *ngIf="version.referenceType === 'BRANCH'" class="fa fa-code-fork" aria-hidden="true" title="Branch"></i>
-            <i *ngIf="version.referenceType === 'TAG'" class="fa fa-tag" aria-hidden="true" title="Tag"></i>
-            <span
-              *ngIf="version.commitID && (version.commitID | commitUrl: workflow?.providerUrl); else noCommitID"
-              [tooltip]="version.commitID ? 'Commit ID: ' + version.commitID : ''"
-            >
-              &nbsp;<a [href]="version.commitID | commitUrl: workflow?.providerUrl">{{ version?.name }}</a>
-            </span>
-            <ng-template #noCommitID
-              ><span>{{ version?.name }}</span></ng-template
-            >
-          </td>
-          <td *ngIf="workflow?.mode === WorkflowType.ModeEnum.HOSTED">{{ version?.versionEditor?.name }}</td>
-          <td>{{ version.last_modified | date: 'medium' }}</td>
-          <td>
-            <mat-icon *ngIf="version.valid">check</mat-icon>
-            <mat-icon *ngIf="!version.valid">close</mat-icon>
-          </td>
+        <a
+          href="https://doi.org/{{ version.doiURL }}"
+          *ngIf="version.doiURL"
+          data-cy="workflow-version-DOI-badge"
+          matTooltip="Click to view this version's DOI entry information."
+          target="_blank"
+          rel="noopener noreferrer"
+          ><img src="{{ zenodoUrl }}/badge/DOI/{{ version.doiURL }}.svg" alt="DOI"
```

Comment:
It was like this and super minor and I wouldn't necessarily bother, but I think the alt should contain the DOI URL as well.